doc/source/tutorial/neighborfinding.rst

```diff
@@ -0,0 +1,217 @@
+As a result, the computations involve performing pair calculations.
+Although in principle such calculations should typically be conducted for all possible pairs, it is usually sufficient to perform analyses only on particles within a given distance that characterizes the local environment.
+This requirement is analogous to the force calculations typically performed in molecular dynamics simulations, where a cutoff radius is specified beyond which pair forces are assumed to be small enough to neglect.
+
+
+Neighbor Querying
+=================
+
+To support efficiently finding neighbors in periodic systems, the **freud** library provides a unified interface to finding neighbors that can both be accessed directly by users and is leveraged internally by all methods that require such calculations.
+The central interface for neighbor finding is the :py:class:`freud.locality.NeighborQuery` family of classes, which provide methods for dynamically finding neighbors given a :py:class:`freud.box.Box`.
+The :py:class:`freud.locality.NeighborQuery` class defines an abstract interface for neighbor finding that is implemented by its subclasses, namely the :py:class:`freud.locality.LinkCell` and :py:class:`freud.locality.AABBQuery` classes.
+These classes represent data structures used to accelerate neighbor finding.
+
+In general, these data structures operate by constructing them using one set of points, after which they can be queried to efficiently find the neighbors of arbitrary other points.
+The queries can either be based stricly on a distance cutoff (using :py:class:`freud.locality.NeighborQuery.queryBall`), or by searching for a specific number of neighbors (using :py:class:`freud.locality.NeighborQuery.query`).
+These classes expose a standardized API for this task, returning a generator than can be easily looped over.
+Alternatively, the pairs of points can be converted into a :py:class:`freud.locality.NeighborList`, a lightweight class that can be used to store pairs of bonds that will be used multiple times.
+For example, the following code first uses the :py:class:`freud.locality.LinkCell` to search for at least 6 neighbors for all points in a given set and calculate the average distances between them, and then uses a :py:class:`freud.locality.AABBQuery` to create a :py:class:`freud.locality.NeighborList` based on a distance cutoff:
+
+.. code-block:: python
+
+    import numpy as np
+    import freud
+
+    # As an example, we randomly generate 100 points in a 10x10x10 cubic box.
+    L = 10
+    num_points = 100
+
+    # Note that we are shifting all points into the expected range for freud.
+    # An alternative would be to simply wrap all points back into the box using
+    # the box.wrap method.
+    points = np.random.rand(num_points)*L - L/2
+    box = freud.box.Box.cube(L)
+
+    # Linked cell list are parameterized by the size of the individual cells.
+    # Intelligent choices of the cell width are crucial for good performance.
```

Comment:
It would be kinder to say "The choice of cell width has a significant impact on performance." or "For optimal performance, the cell width must be chosen carefully."